### DISCUSSION TITLE
Algorithm/Optimisation/Greedy-Algorithm

### DISCUSSION THREAD

**Discussion by giscus** [xy-241/CS-Notes#3] [General]
# DSA/Algorithms/Greedy-Algorithm

Abstract § Note 向日葵朝着太阳转动， 时刻都在寻求当前汲取最大养分的可能。 最优子结构就是追随太阳最高效的移动轨迹。 移除重复子问题让向日葵紧跟太阳的脚步。 贪心策略在一轮轮的简单选择中，逐步导向最佳的答案。 Used to solve Optimisation Problem Usually better performance than Backtracking &amp; Dynamic Programming Think about what is the Local Optimal Solution.

https://notes.yxy.ninja/Algorithm/Optimisation/Greedy-Algorithm

**Comment by FooNicholas** [comment]
hello
